src/ecs/explorer/ecsContainerNode.ts

```diff
@@ -43,4 +41,18 @@ export class EcsContainerNode extends AWSTreeNodeBase {
     public describeTasks(tasks: string[]) {
         return this.ecs.describeTasks(this.clusterArn, tasks)
     }
+
+    private async hasRunningTasks(): Promise<boolean> {
+        const tasks = await this.ecs.listTasks(this.clusterArn, this.serviceName, TASK_STATUS_RUNNING)
+        return tasks.length > 0
```

Comment:
If this is only an existence check then we don't need more than one result. Maybe it's worth specifying pageSize=1 in the API call.

Reply:
Sure, I can add a `maxResults` parameter to the call.